compiler/onecc-docker/debian/rules

```diff
@@ -2,4 +2,8 @@
 
 %:
 	dh $@
-
+	
+overrid_dh_fixperms:
```

Comment:
```suggestion
override_dh_fixperms:
```

Is the `onecc-docker` file in `/usr/share/one/bin/` when this function is executed?

Reply:
@jyoungyun 

Oh, thank you for your advice. As I looked it up, the permission adjustment to a occurs when the package is installed, so I set it as path `debian/onecc-docker/usr/share/one/bin/onecc-docker`. The path of onecc-docker file was set considering the entry path of the Debian package. I tested it after that, but I give permission to onecc-docker well, so can you check if the code is okay?